src/Constants.sol

```diff
@@ -122,16 +126,16 @@ abstract contract Constants {
   string internal constant USDC_SYMBOL = "USDC";
   string internal constant WETH_SYMBOL = "WETH";
 
-  string internal constant PT_USDC_NAME = "PoolTogether Prize USD Coin";
-  string internal constant PT_USDC_SYMBOL = "PTUSDC";
+  string internal constant PT_USDC_NAME = "Prize USDC - Aave";
```

Comment:
Prize USDC.e - Aave

Reply:
Fixed in this commit: https://github.com/GenerationSoftware/pt-v5-mainnet/pull/3/commits/dfcecebb78e3cace69813181beb20098400cb326